tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,42 @@
+import pdb
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+from gaiatest.apps.contacts.regions.settings_form import SettingsForm
+from gaiatest.apps.contacts.regions.contact_form import ContactForm
+
+class TestContactsRTL(GaiaImageCompareTestCase):
+   
+    def test_contacts_app(self):
+
+        contacts = Contacts(self.marionette)
+        contacts.launch()
+        self.take_screenshot('main')
+        #pdb.set_trace()
+        contacts.tap_settings()
+        self.take_screenshot('main')
+        settings_form = SettingsForm(self.marionette)
+        settings_form.tap_import_contacts()
+        self.take_screenshot('main')
+        settings_form.tap_import_from_sim()
+        self.take_screenshot('main')
+        settings_form.tap_back_from_import_contacts()
+        self.take_screenshot('main')
+        settings_form.tap_order_by_last_name()
+        self.take_screenshot('main')
+        settings_form.tap_done()
+        self.take_screenshot('main')
+        for i in range(0, 2):
+            GaiaImageCompareTestCase.scroll(self.marionette, 'down', contacts.screen_element.size['height'], screen=contacts.screen_element)
+            self.take_screenshot('main')
+
+        contacts.tap_new_contact()
+        self.take_screenshot('main')
+        contact_form = ContactForm(self.marionette)
```

Comment:
yup, NewContact is a derived class of Contactform.  and the thing with python is that you don't need to import classes that you're indirectly instantiating- so if you get the class object this way, you don't need to import *forms classes. It's strange but that's how python works it seems.